src/components/Garden/ProposalDetail/ProposalDetail.js

```diff
@@ -64,6 +65,7 @@ import { ZERO_ADDR } from '@/constants'
 // assets
 import warningIcon from '../Agreement/assets/warning.svg'
 import { AbstainCardHeader } from '../Feed/AbstainCard'
+import LoadingRing from '@/components/LoadingRing'
```

Comment:
wondering why we removed the loadingRing component for the loading component and then we added the loadingRing component again donwn there

Reply:
I was trying the new Loading and add it again. Rolling back that change.